openedx_rocketchat/static/js/rocket_chat_discussion.js

```diff
@@ -0,0 +1,75 @@
+
+class RocketChatNavigation {
+
+    constructor(url_change_role) {
+        this.pageContentMain = $(".page-content-main");
+        this.pageContentSecondary = $(".page-content-secondary");
+        this.url_change_role = url_change_role;
+
+        this.loadComponents(this.pageContentMain, this.pageContentSecondary);
+
+        $(window).bind('hashchange', $.proxy(function(){
+            this.loadComponents(this.pageContentMain, this.pageContentSecondary);
+        },this));
+
+        $(".change-role-rocketchat").unbind().click($.proxy(function() {
+            this.change_role(this.url_change_role);
+        }, this))
+    }
+
+
+    change_role(url_change_role){
+        var input = $(".rocket-chat-user-input")
+        var username = input.val();
+        var role = "coach";
+        $.ajax({
+            type: "POST",
+            url: url_change_role,
+            data: {"username": username, "role": role},
+            success: function(){
+                input.val("");
+                alert("The role has been changed");
+            },
+            error: function (request, status, error) {
```

Comment:
If you are going to use a space after a function declaration, make sure that other declarations have the same syntax. @andrey-canon

Reply:
ok